<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTrendChartGenDlg</class>
 <widget class="QWidget" name="QTrendChartGenDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>39</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>背 景 色：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_noedit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Read_Address">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>271</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>读  取 地 址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>69</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数  据 类 别：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Read_Address_Keybord">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>编    号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DataType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>69</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Read_Triggre_Keybord">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>读取之触发位：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Read_Triggre">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>220</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="Number_Data">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>220</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>曲 线 条 数：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="R_FromLeft">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>从左到右</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>每组数据点数：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Points_Per_Data">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>250</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="Sampling_Time">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>300</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>采  样 时 间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>300</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>x1s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>方        向：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="R_FromRight">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>从右到左</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="If_Show_Data_Point">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>画点标记</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="Show_Connected_Line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>画连接线</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>底        色：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Clear_Trigged_Keybord">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>清除之触发位：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Clear_Trigged">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BackColor">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>39</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BG_Color">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>检测</string>
   </property>
   <widget class="QCheckBox" name="m_ifuse">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>启用</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_colorlable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜    色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_checkcolor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="m_addrlable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>检测地址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_checkaddredit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_checkbtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="Transparence">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>透明度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Per_Data_Variable_lineEdit">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>310</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Per_Data_Variable_keyboardBtn">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>310</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="If_Per_Data_Variable">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>每组数据点数可变：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bReadAutoOff">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自动复位</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bClearAutoOff">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>476</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自动复位</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="If_Use_Control_Add">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>是否使用控制地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Control_Add_Keyboard">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>注意：该地址只能为内部地址</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Control_Add">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>注意：该地址只能为内部地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>控制地址：</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_noedit</tabstop>
  <tabstop>Transparence</tabstop>
  <tabstop>DataType</tabstop>
  <tabstop>m_checkaddredit</tabstop>
  <tabstop>m_checkbtn</tabstop>
  <tabstop>Read_Triggre</tabstop>
  <tabstop>Read_Triggre_Keybord</tabstop>
  <tabstop>bReadAutoOff</tabstop>
  <tabstop>Number_Data</tabstop>
  <tabstop>Points_Per_Data</tabstop>
  <tabstop>Read_Address</tabstop>
  <tabstop>Read_Address_Keybord</tabstop>
  <tabstop>Sampling_Time</tabstop>
  <tabstop>If_Per_Data_Variable</tabstop>
  <tabstop>Per_Data_Variable_lineEdit</tabstop>
  <tabstop>Per_Data_Variable_keyboardBtn</tabstop>
  <tabstop>Clear_Trigged</tabstop>
  <tabstop>Clear_Trigged_Keybord</tabstop>
  <tabstop>bClearAutoOff</tabstop>
 </tabstops>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
